website/docs/components/rich-tooltip/partials/code/how-to-use.md

```diff
@@ -0,0 +1,287 @@
+
+```handlebars
+<Hds::Text::Body @tag="p" @size="300">
+  Lorem
+  <Hds::RichTooltip @isInline={{true}} as |RT|>
+    <RT.ToggleInfoText>ipsum dolor</RT.ToggleInfoText>
+    <RT.PopoverContent>
+      <Hds::Text::Display @tag="h4" @size="200">Some title</Hds::Text::Display>
+      <Hds::Text::Body @tag="p" @size="200">Descriptive information</Hds::Text::Body>
+    </RT.PopoverContent>
+  </Hds::RichTooltip>
+  sit amet consectetur adipiscing elit.
+</Hds::Text::Body>
+```
+
+Similarly, you can apply a typographic class, or a custom CSS class, to a parent container.
+
+Which one of these approaches to use will depend on the context where the code is implemented, so we leave it to the consumers to decide which approach works better for them.
+
+#### Interactivity
+
+By default the visibility of the tooltip is toggled via "soft" event listeners (hover/focus) applied to the "toggle" container. _Notice: from a purely technical standpoint, the events are `mouseEnter/Leave` and `focusIn/Out`._
+
+You can change this behavior and opt for a more explicit user interaction enabling "click" events (in this case the "soft" events are disabled) setting the argument `@enableClickEvents` to `true`:
+
+```handlebars
+<Hds::RichTooltip @enableClickEvents={{true}} as |RT|>
+  <RT.ToggleInfoText @size="medium" @icon="info">More info</RT.ToggleInfoText>
+  <RT.PopoverContent>
+    <Hds::Text::Display @tag="h4" @size="200">Some title</Hds::Text::Display>
+    <Hds::Text::Body @tag="p" @size="200">Descriptive information</Hds::Text::Body>
+  </RT.PopoverContent>
+</Hds::RichTooltip>
+```
+
+Independent of which interaction is used, the tooltip can be dismissed by clicking outside of the component or with the `esc` key (this "light dismiss" behaviour is automatically provided by the [Popover API](https://developer.mozilla.org/en-US/docs/Web/API/Popover_API/Using#auto_state_and_light_dismiss)).
+
+#### Placement
+
+By default the tooltip is shown below the toggle, visually centered. It's possible to change the initial position of the tooltip using the `@popoverPlacement` argument:
+
+```handlebars
+<Hds::RichTooltip @popoverPlacement="top-start" as |RT|>
+  <RT.ToggleInfoText @size="large" @icon="info">Lorem ipsum</RT.ToggleInfoText>
+  <RT.PopoverContent>
+    <Hds::Text::Display @tag="h4" @size="300">Lorem ipsum</Hds::Text::Display>
+    <Hds::Text::Body @tag="p" @size="200">Sit amet consectetur ut ultrices id venenatis in felis auctor ante.</Hds::Text::Body>
+  </RT.PopoverContent>
+</Hds::RichTooltip>
+```
+
+#### Collision detection
+
+The RichTooltip component automatically adapts its alignment depending on its position in relation to the viewport, to avoid "collisions" with the browser window boundaries. This means that when an end-user scrolls the page, or resizes the browser, the position of the tooltip on the page dynamically adapts to these changes (along the two axes, main and secondary).
```

Comment:
[suggestion]
```suggestion
The RichTooltip component automatically adapts its alignment depending on its position relative to the viewport to avoid "collisions" with the browser window boundaries. This means that when an end-user scrolls the page, or resizes the browser, the position of the tooltip on the page dynamically adapts to these changes (along the two axes, main and secondary).
```